### PR TITLE
PSLab v2.0.15

### LINKED ISSUE #1811: Release PSLab v2.0.15
**Actual Behaviour**

Development branch has enough commits and new features to make a new release

**Expected Behaviour**

Release v2.0.15 with version code 16

**Steps to reproduce it**

> Not available

**LogCat for the issue**

> Not available

**Screenshots of the issue**

> Not available

**Would you like to work on the issue?**

@neel1998 can try.
Steps:
1. Increment version code and version in gradle file
2. Make a PR with the change
3. Create a merge request to master branch

### PR DESCRIPTION
**Pull request by neel1998**
Fixes #1811 

**Changes**: version upgraded

@mariobehling  @CloudyPadmal please check

### CODE REVIEW

**Comment by neel1998** [comment]
@CloudyPadmal @mariobehling please review

File: app/build.gradle [hunk 1]
```diff
@@ -8,8 +8,8 @@ android {
         applicationId "io.pslab"
         minSdkVersion rootProject.ext.minSdkVersion
         targetSdkVersion rootProject.ext.targetSdkVersion
-        versionCode 15
-        versionName "2.0.14"
+        versionCode 16
+        versionName "2.0.15"
```
Comment (CloudyPadmal) [inline app/build.gradle]: Send a PR with only this change. Discard the rest.

**Comment by CloudyPadmal** [comment]
@neel1998 I'll brief you on the release process.
1. Make a PR with version code incremented
2. Once it is merged and the build is passed, make a merge request from development branch to master branch. You'll have to update the branch here
3. Once that build is successful, we'll make a "Create a merge commit" and then the dev branch will be merged to master and apk files will be automatically build with a signed APK sent to Google Play Store.
4. I'll walk you through Google Play releases later and you can search online how apps are released in Google Play. Only difference with us is that we have an APK already waiting to be published there. No need to upload anything. For now we'll go up to 3.

File: app/build.gradle (hunk 1 above)
Comment (neel1998) [reply app/build.gradle]: So should I make a PR to development branch?

**Comment by neel1998** [comment]
@CloudyPadmal  so should i close this PR and make anew one to development branch with just gradle changes?

**Comment by CloudyPadmal** [comment]
No, you can reset the branch to upstream/development and make the changes in gradle file and make a force push

**Comment by neel1998** [comment]
@CloudyPadmal  i did the following 
```
git reset upstream/development
git add .
git commit 
git push origin +<Branch_name>
```
But i think since i made the PR to master branch all other commits are included...

**Comment by CloudyPadmal** [comment]
`git reset --soft upstream/development`
and not `git add .` only add the changes

**Comment by neel1998** [comment]
@CloudyPadmal i did what you asked. Is this correct?... I think still all the commits are there in PR bcz the PR is made to master

**Comment by CloudyPadmal** [comment]
Try reset --hard

**Comment by CloudyPadmal** [comment]
@neel1998 any update? This would take only 5 minutes.

**Comment by neel1998** [comment]
@CloudyPadmal  i tried reset -- hard as well. I think i should open a new PR to dev branch

**Comment by CloudyPadmal** [comment]
Ok, please do so. This is basic git understanding.

**Pull request closed without merging**